packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -31,3 +31,7 @@ export const userLogPayloadGuard = z.object({
 });
 
 export type UserLogPayload = z.infer<typeof userLogPayloadGuard>;
+
+export const connectorConfigGuard = z.object({});
```

Comment:
Add a TODO comment?

Reply:
`// TODO ` (with trailing space) works too.

Reply:
didnt see the lint complaint in GitHub, would you like to add an issue to see what happened? should be able to see the warning here.

Reply:
Change to `// TODO:` still not working, created LOG-1289